linker/shared/src/main/scala/org/scalajs/linker/analyzer/Analyzer.scala

```diff
@@ -377,7 +377,10 @@ private class AnalyzerRun(config: CommonPhaseConfig, initial: Boolean,
 
       if (state eq loading) {
         // We just added `loading`, actually load.
-        val maybeInfo = infoLoader.loadInfo(className)
+        val maybeInfo = infoLoader.loadInfo(
+          className,
+          config.coreSpec.linkTimeProperties
```

Comment:
Let's give that argument at the construction of the `infoLoader`. The `infoLoader` caches computed infos. That is not sound if we can give a new link-time configuration every time we call `loadInfo`.